omod/src/main/java/org/openmrs/module/coreapps/page/controller/MarkPatientDeadPageController.java

```diff
@@ -38,6 +41,13 @@ public void get(@SpringBean PageModel pageModel, @RequestParam(value = "breadcru
         if (conceptId != null && !conceptId.contains("[a-zA-Z]+")) {
             pageModel.put("conceptAnswers", getConceptAnswers(Integer.parseInt(conceptId)));
         }
+        List<Visit> visits = Context.getVisitService().getVisitsByPatient(patient);
+
+        if(visits.size() > 0){
```

Comment:
The assumption that the first record has the last visit date, is an internal implementation detail. Not part of the public interface. Therefore, can you, at least add a comment about this assumption, for future maintainers?

Reply:
I added an inline comment for this